app/src/main/java/com/sopt/now/compose/ui/home/HomeViewModel.kt

```diff
@@ -1,77 +1,67 @@
+import com.sopt.now.compose.data.ServicePool
+import com.sopt.now.compose.data.dto.response.ResponseFriendDto
+import com.sopt.now.compose.data.dto.response.ResponseUserDto
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
 class HomeViewModel : ViewModel() {
-    val mockFriendList = listOf(
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "이의경",
-            phone = "010-1234-5678",
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "공세영",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "김명석",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "김아린",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "김언지",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "배지현",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "박동민",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "박효빈",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "배찬우",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "손민재",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "송혜음",
-            phone = "010-1234-5678"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "유정현",
-            phone = "010-5678-1234"
-        ),
-        Friend(
-            profileImage = R.drawable.img_main_profile,
-            name = "이가을",
-            phone = "010-5678-1234"
-        )
-    )
+    val userData: MutableState<ResponseUserDto.UserData?> = mutableStateOf(null)
+    val friendList: MutableState<List<Friend>> = mutableStateOf(emptyList())
 
-    fun getFriendData(): List<Friend> = mockFriendList
-}
\ No newline at end of file
+    fun getUserData(userId: String) {
+        ServicePool.userService.getUser(userId).enqueue(object : Callback<ResponseUserDto> {
+            override fun onResponse(
+                call: Call<ResponseUserDto>,
+                response: Response<ResponseUserDto>
+            ) {
+                if (response.isSuccessful) {
+                    val data: ResponseUserDto? = response.body()
+                    userData.value = data?.data
```

Comment:
```suggestion
                    userData.value = response.body()?.data
```
data를 사용하는 곳이 해당 부분 밖에 없다면 바로 넣어줘도 괜찮지 않을까요?